readthedocsext/theme/templates/allauth/layouts/base.html

```diff
@@ -2,6 +2,13 @@
 
 {% block content-wrapper %}
   <div class="ui very padded container">
```

Comment:
Not entirely sure, but the HTML structure is the same up until this block. We might be able to just move the override to `content` block instead. This requires removing a closing div below too

```suggestion
{% block content %}
```

Reply:
@agjohnson This template is also defining the content inside the wrapper a few lines down, so I think we still need the wrapper here.